src/uploaders/txc_uploader/txc_processor.py

```diff
@@ -242,7 +242,52 @@ def insert_into_txc_operator_service_table(cursor, operator, service, line, regi
             return None
 
         raise
-        
+
+
+def check_txc_line_exists(cursor, operator, service, line, data_source, cloudwatch, logger):
+    (
+        noc_code,
+        start_date,
+        operator_short_name,
+        service_description,
+        service_code
+    ) = extract_data_for_txc_operator_service_table(operator, service)
+
+    query = f"""
+        SELECT id FROM txcOperatorLine
+        WHERE nocCode = %s AND lineName = %s AND serviceCode = %s AND startDate = %s AND dataSource = %s
+        LIMIT 1
+    """
+
+    cursor.execute(
+        query,
+        [
+            noc_code,
+            line['LineName'],
+            service_code,
+            start_date,
+            data_source
+        ]
+    )
+    result = cursor.fetchone()
+    operator_service_id = None
+    
+    if result and len(result) > 0:
+        operator_service_id = result[0]
+
+
+
+    if operator_service_id:
+        logger.info("Existing line found - '{}' - '{}' - '{}' - '{}' - '{}'".format(
```

Comment:
Could this become:
```
logger.info(f"Existing line found - '{noc_code}' - '{line['LineName']}' - '{service_code}' - '{start_date}' - '{data_source}'")
```

Reply:
Yeah, this needs doing across all the string templates really